Capstone/Feed/HomeViewController.m

```diff
@@ -0,0 +1,112 @@
+//
+//  HomeViewController.m
+//  Capstone
+//
+//  Created by Abel Kelbessa on 7/4/22.
+//
+
+#import "HomeViewController.h"
+#import "HomeCell.h"
+#import "UIImageView+AFNetworking.h"
+#import "DetailsViewController.h"
+#import "APIManager.h"
+
+@interface HomeViewController ()<UITableViewDelegate, UITableViewDataSource, UISearchBarDelegate>
+
+@property(strong, nonatomic) NSArray<NSDictionary *> *colleges;
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
+@property (strong, nonatomic) NSMutableArray *collegeDictionaryArrays;
+@property (weak, nonatomic) IBOutlet UISearchBar *searchBar;
+@property (strong, nonatomic) NSMutableArray *filteredColleges;
+
+@end
+
+bool isFiltered;
+
+@implementation HomeViewController
+
```

Comment:
For private instance variables like this in Obj-C, you'll want to do something like this instead:

```suggestion
@implementation HomeViewController {
   bool _isFiltered;
}
```

Otherwise, you might have a global variable.

On second thought, for the non-outlets, you might want to also do the same:

```suggestion
@implementation HomeViewController {
   bool _isFiltered;
   NSArray<NSDictionary *> *_colleges;
   NSMutableArray<NSDictionary *> *_collegeDictionaryArrays;
   NSMutableArray<NSDictionary *> *_filteredColleges;
}
```